crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -26,7 +32,7 @@ impl TestContractCommand {
 				style(" Pop CLI ").black().on_magenta()
 			))?;
 
```

Comment:
Unnecessary newlines in this function. Newlines should in my opinion only have to be used in big code blocks and separate processes that happen. An `if else` statement is in my opinion one process.